cache/cache.go

```diff
@@ -1,11 +1,47 @@
 package cache
 
-import "time"
+import (
+	"time"
+
+	aw "github.com/deanishe/awgo"
+)
 
 const (
 	maxCacheAgeMinutes = 1 // How long to cache data
 )
 
+type BaseCache struct {
+	wf *aw.Workflow
+}
```

Comment:
Define the BaseCache struct to share the basic cache codes with the other cache structs.